packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/sdk/test/arm-test/review/arm-networkanalytics.api.md

```diff
@@ -343,6 +343,13 @@ export enum KnownActionType {
     Internal = "Internal"
 }
 
+// @public
+export enum KnownAzureClouds {
+    AZURE_CHINA_CLOUD = "AZURE_CHINA_CLOUD",
+    AZURE_PUBLIC_CLOUD = "AZURE_PUBLIC_CLOUD",
+    AZURE_US_GOVERNMENT = "AZURE_US_GOVERNMENT"
```

Comment:
@xiangyan99 I'd like to confirm the enum words with you since I notice we have AZURE_CHINA_CLOUD and AZURE_PUBLIC_CLOUD but not AZURE_US_GOVERNMENT_CLOUD(no "_CLOUD" suffix), is this by design?

Reply:
Let's align with other languages.